android/src/main/java/com/vanethos/notification_permissions/NotificationPermissionsPlugin.java

```diff
@@ -47,7 +47,7 @@ public void onMethodCall(MethodCall call, MethodChannel.Result result) {
           result.error(call.method, "context is not instance of Activity", null);
         }
       } else {
-        result.success(null);
+        result.success(PERMISSION_GRANTED);
```

Comment:
This was returning always null (`unknown`) when it should return `granted`. The same happens when it's `denied`.

Reply:
Yikes, how did I miss this? thank you so much :pray: